src/api.js

```diff
@@ -20,9 +20,23 @@ export const getConferenceById = async (id) => {
   }
 };
 
-export const createNewConference = async (conference) => {
+export const createNewConference = async conference => {
   try {
-    const response = await axios.post(`http://localhost:8080/conferences, ${conference}`);
+    const response = await axios.post(
+      `http://localhost:8080/conferences, ${conference}`
```

Comment:
should probably be written as `post('http://localhost:8080/conferences', conference)`